spec/models/loader_orchestrate_spec.rb

```diff
@@ -127,4 +127,24 @@ def verify_data(path)
       end
     end
   end
+
+  context "with non-root user policy" do
+    let(:base_policy_path) { 'empty.yml' }
+    it "loads the minimal policy" do
+      expect(resource_policy).to be
+      verify_data 'base/empty.txt'
```

Comment:
What's happening here with this check?  If I'm reading it correctly, let's get parentheses to improve readability:

```ruby
expect(resource_policy).to be verify_data('base/empty.txt')
```

Reply:
removed this code block - it is not nessesary